### ISSUE TITLE
[Suggestion]  Friendly map summons sprites catalog

### ISSUE THREAD

**Issue by fusiletaw** [Aceship/AN-EN-Tags#106]
Can you make display page for friendly map summons sprite such as episode 8's guerilla shieldguards, SP charger in kazimierz maps, and waterproof coating device in DHS event?

**Comment by Aceship** [comment]
i think its in the enemy page too, its still considered as enemy 

![image](https://user-images.githubusercontent.com/5337323/129543560-39f4b585-a936-4635-a61e-0880e7abc1be.png)

**Comment by Aceship** [comment]
or is it the different one 🤔

**Comment by fusiletaw** [comment]
yes... i mean friendly objects like guerillas. not like civilian

**Comment by tmchansen** [comment]
Would the jammers like used in 5-6 fit in with what you're talking about? I'm not sure where they would fit. They're pretty situational and typically have  "how to use" missions right before they're usable if so.
